flybywiresim/aircraft
# fix(mcdu): change page title of init b

Fixes #[issue_no]

## Summary of Changes
Based on observations of some A320neo cockpit documentations, reading the FCOM and the Honeywell Pegasus the INIT B page title is change dynamically based on the fuel planning state.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

1. Setup INIT A page
2. Switch to INIT FUEL PRED page.
3. Add ZFW/ZFWCG
4. Press LSK3R to start FUEL PLANNING
5. Page title should change to "INIT FUEL PLANNING"
6. Finish FUEL PLANNING and press LSK3R "BLOCK CONFIRM"
6. INIT B page title should now be and stay "INIT FUEL PRED"


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
